examples/ERC721/jay-small-test.mjs

```diff
@@ -0,0 +1,110 @@
+import { loadStdlib, util } from '@reach-sh/stdlib';
```

Comment:
This file was Nevers' original test. I think it should be deleted and instead we should only test with the general 721 test suite